corehq/apps/users/models.py

```diff
@@ -1725,12 +1725,12 @@ def save(self, fire_signals=True, spawn_task=False, **params):
             log_signal_errors(results, "Error occurred while syncing user (%s)", {'username': self.username})
             sync_usercases_if_applicable(self, spawn_task)
 
-    def delete(self, deleted_by, deleted_via=None):
+    def delete(self, domain, deleted_by, deleted_via=None):
```

Comment:
Just to confirm,
This is because a "domain" would be present on the `CommCareUser` itself which it is bound to.

Just want to confirm if things are now different because of Mirrored domains? not sure if that is the right term to put it. Is it possible for a domain to delete a commcare user bound to a different domain?

Reply:
It either is or likely will soon be possible for one domain to delete a different domain's mobile worker, yes.

Reply:
cool. Thanks @orangejenny 

@snopoke let me know if it sounds reasonable now to pass along domain for both mobile and web users.